ccm_web/server/src/api/api.admin.handler.ts

```diff
@@ -113,6 +114,9 @@ export class AdminApiHandler {
     )
     const coursesWithSectionsResult = await Promise.all(coursesWithSectionsApiPromises)
     const finalResult = makeResponse<CourseWithSections>(coursesWithSectionsResult)
+    if (!isAPIErrorData(finalResult)) {
+      logger.debug(`Number of sections returned: ${finalResult.length}`)
+    }
```

Comment:
The log message changes in this file are not directly related to the issue, but I think they are somewhat useful in diagnosing the issue we're working on that also prompted the configuration change. I can remove them if you'd like.